Project/zepeto-modules/Assets/Zepeto ScreenShot Module/ZepetoScript/ScreenShotModeManager.ts

```diff
@@ -15,9 +15,10 @@ export default class ScreenShotModeManager extends ZepetoScriptBehaviour {
     public selfieCameraPrefab: GameObject;
     private selfieCamera: Camera;
     private zepetoCamera: Camera;
+
+    public additionalRunSpeedInSelfieMode: number = 0;
+    public additionalWalkSpeedInSelfieMode: number = 0;
```

Comment:
Plus, if we set the `additionalRunSpeedInSelfieMode` to 5, the character won't move if we get into the screenshot UI, with default camera mode.

Reply:
Hi, @bohyunjung ! Thank you for your comments. This variable was left out for adjusting run & walk speed in selfie mode. But since the default character's speed can vary based on user-custom settings, setting a default value might lead to unintended behaviors. So i initialized it to 0. I'll think about how i can clarify this further.

Reply:
> Plus, if we set the `additionalRunSpeedInSelfieMode` to 5, the character won't move if we get into the screenshot UI, with default camera mode.

This issue seems to be related to initialization. i'll check it!